src/java.base/share/man/java.md

```diff
@@ -1020,6 +1020,11 @@ These `java` options can be used to enable other advanced options.
 
 These `java` options control the runtime behavior of the Java HotSpot VM.
 
+`-XX:+UseCompactObjectHeaders`
+:   Enables compact object headers. By default, this option is disabled.
+    Enabling this option reduces memory footprint in the Java heap by
+    4 bytes per object (on average).
```

Comment:
The options are (with 3 exceptions) in alphabetical order - please move this to  where the other `UseXXX` flags are.

This does not explain at all why someone would want to use, or not use, COH. What are the trade-offs for turning it on? Please explain the usage.

Reply:
If the flag is off by default but apparently only has the benefit of reducing footprint, then the reader will wonder why it is off by default. At the moment this reads like we have a flag that will benefit everyone and yet we expect them to turn it on themselves because for some reason we chose not to. If this is really a no-lose situation we can say something like I did in the Release Note JBS issue i.e. something like "this is a good thing to do but still under evaluation so is off by default for now".

Reply:
Yes this is really a bit odd. 'Still under evaluation' is also weird considering that we just removed the 'experimental' status, which basically saying 'still under evaluation'. Anyway, I will adopt your wording from the release note and move the text down as you suggested.